.circleci/config.yml

```diff
@@ -953,6 +953,15 @@ jobs:
                 when: always
                 command: |
                   ./Tests/scripts/install_content_and_test_integrations.sh "$INSTANCE_ROLE"
+            - run:
+                name: Validate Premium Packs
+                shell: /bin/bash
+                when: always
```

Comment:
If installation failed - is there a point running this one?

Reply:
I think yes. It will check the index.json in the cloud which is good.